R/timer.R

```diff
@@ -86,6 +88,40 @@ TimerCallbacks <- R6Class(
   )
 )
 
+TimerCallbacks <- R6Class(
```

Comment:
Is this class necessary? It appears to do exactly the same thing as the superclass (except that you can't pass in anything to the `initialize` method).

Reply:
Fair point. I had in my head that I wanted to keep the contract for TimerCallbacks consistent. But I now realize that it's only internal, so who cares. I'll simplify.